scripts/calculateRewards/proofOfImpact.ts

```diff
@@ -133,7 +106,6 @@ async function main(args: ReturnType<typeof parseArgs>) {
     'rewards:',
     rewards.map((r) => ({
       referrerId: r.referrerId,
-      kpi: r.kpi,
```

Comment:
nit: I liked having this info to as a quick sanity check to see if the reward looks plausible compared to the KPI.